homeassistant/components/maico/maico.py

```diff
@@ -0,0 +1,89 @@
+"""Maico Data."""
+from __future__ import annotations
+
+import logging
+
+import httpx
+import xmltodict
+
+from .connection import Connection
+
+# from .sensor import Sensor
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Maico:
+    """Maico Data Class."""
```

Comment:
I believe the standard would be that this sort of class is in a separate PyPI library, but I'll defer to someone more HA core on that.